tests/test_install.py

```diff
@@ -355,6 +355,16 @@ async def run_test(selenium, url, wheel_name):
     run_test(selenium_standalone_micropip, snowball_wheel.url, snowball_wheel.filename)
 
 
+def test_emfs_error(selenium_standalone_micropip):
+    @run_in_pyodide()
+    async def run_test(selenium):
+        import micropip
+
+        await micropip.install("emfs:a-2.0.2-cp312-cp312-pyodide_2024_0_wasm32.whl")
+
+    run_test(selenium_standalone_micropip)
```

Comment:
Probably this test should check the error message?

Reply:
Yeah it is failing now. I am having trouble running the tests locally so I figured I would let the CI tell me.